index.html

```diff
@@ -82,7 +82,11 @@
               response[i] = [];
             }
 
-            ws = $websocket('ws://127.0.0.1:8000/mtr');
+            var url = new URL(window.location.href);
+            url.protocol = url.protocol.replace('http', 'ws');
+            url.href = url.href + "mtr"
+            ws = $websocket(url.href);
+            //console.log("log: " + url.href);
```

Comment:
I like this protocol replacement! But can you please remove the commented out `console.log()` line?